scripts/install-tree-sitter-lib

```diff
@@ -53,7 +53,14 @@ fi
   cd tree-sitter
   git clean -dfX
   make
-  sudo make install
+
+  # When building from source on homebrew sudo is not allowed
+  # but also not needed
+  if [[ -z "${HOMEBREW_SYSTEM}" ]]; then
+    sudo make install
+  else
+    make install
+  fi
```

Comment:
`[[ -z "${HOMEBREW_SYSTEM+x}" ]]` is the way to go here (works on old versions of bash like the default version on macos).